### PR TITLE
[AUTHZ] Check Authz plugin's spec json files in UT

### LINKED ISSUE #4715: [Improvement] [AUTHZ] Spec json files should be auto generated, rather than edit manually
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/kyuubi/issues?q=is%3Aissue) and found no similar issues.

### What would you like to be improved?

In #4658, this pr edit the [table_command_spec.json](https://github.com/apache/kyuubi/pull/4660/files#diff-9e9de8073ec13128987f6c7e163f71696771bebb96442ed5808642e2c8453cec), but this file should be generated by JsonSpecFileGenerator.scala, rather than edited this file manually.

### How should we improve?

_No response_

### Are you willing to submit PR?

- [X] Yes. I would be willing to submit a PR with guidance from the Kyuubi community to improve.
- [ ] No. I cannot submit a PR at this time.

### PR DESCRIPTION
**Pull request by packyan**
### _Why are the changes needed?_

to close #4715

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by bowenliang123** [comment]
Hi, the spec json file should not be regenerated in each build, by design. But yes, it should be checked during ci builds.

**Comment by packyan** [comment]
> Hi, the spec json file should not be regenerated in each build, by design. But yes, it should be checked during ci builds.

Yes, I will add a UT to check generated golden files.

**Comment by packyan** [comment]
> Alright, I mean change it from executing the main method to running it in test suites, for both validation in testing and regeneration, just to be clear.

understood

**Comment by bowenliang123** [comment]
Would you like to remove redundant changes in generator and policy json file of  https://github.com/apache/kyuubi/pull/4716 which is already merged ?  @packyan

**Comment by packyan** [comment]
> Would you like to remove redundant changes in generator and policy json file of #4716 which is already merged ? @packyan

Done, wait for CI finished.

File: extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/JsonSpecFileGenerator.scala [hunk 1]
```diff
@@ -18,37 +18,67 @@
...
+      Files.write(
+        filePath,
+        generatedStr.getBytes(StandardCharsets.UTF_8),
+        StandardOpenOption.CREATE,
+        StandardOpenOption.TRUNCATE_EXISTING)
+    } else {
+      val existedFileContent =
+        FileUtils.readFileToString(filePath.toFile, StandardCharsets.UTF_8)
+      // scalastyle:off println
+      println(s"checking ${specArr.length} specs in $filename")
```
Comment (bowenliang123) [inline extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/JsonSpecFileGenerator.scala:74]: Remove this unnecessary print.

**Review by bowenliang123 — APPROVED** [review]
LGTM overall. Minor comment.

**Comment by bowenliang123** [comment]
Thanks for your continuous contribution. Merged to master.

**Pull request closed without merging**